dtm0/it/all2all/all2all

```diff
@@ -0,0 +1,239 @@
+#!/bin/bash
+#set -x
+set -e
+
+MOTR_ROOT=$(realpath ../../..)
+MOTR_ST_UTILS_DIR=${MOTR_ROOT}/motr/st/utils/
+MOTR_VAR_DIR=/var/motr
+TEST_ROOT=$MOTR_VAR_DIR/all2all_test
+CURRENT_CDF=$PWD/test.yaml
+CONFD_XC=/var/lib/hare/confd.xc
+LOOP_IMG_DIR=$TEST_ROOT
+
+M0D_DIR_COMMON=$MOTR_VAR_DIR/m0d-0x7200000000000001:0x
+M0D_IDS=(c 1a 28)
+M0D_PIDS=()
+
+M0D_ENDPOINTS=()
+M0D_SERV_FIDS=()
+
+POOL_WIDTH=4
+IPOOL_WIDTH=2
+IMGS_CNT=$((POOL_WIDTH + IPOOL_WIDTH))
+
+. ${MOTR_ROOT}/scripts/addb-py/chronometry/common/common_funcs
+
+
+function check_hare()
+{
+    local rc
+    set +e
+    which hctl >/dev/null 2>&1
+    rc=`echo $?`
+    if [ $rc -ne 0 ]
+    then
+        _err "Hare is not installed, install it and try again."
+        exit 1
+    fi
+    set -e
+}
+
+function stop_cluster()
+{
+    set +e
+    hctl shutdown || {
+        _warn "Cluster stop FAILED! Trying to go further."
+    }
+    
+    #systemctl reset-failed hare-hax
+    set -e
+}
+
+function bootstrap_cluster()
+{
+    hctl bootstrap --mkfs $CURRENT_CDF
+}
+
+#============= DEVICES SECTION =============#
```

Comment:
this section is not needed,
call from m0setup needed function and do not copy/paste them